lib/install/validate/validator.go

```diff
@@ -320,20 +324,55 @@ func (v *Validator) target(ctx context.Context, input *data.Data, conf *config.V
 	// check if host is managed by VC
 	v.managedbyVC(ctx)
 
+	// check target with ops credentials
 	u := input.Target.URL
 
+	conf.Username = input.OpsUser
+	conf.Token = input.OpsPassword
+	conf.TargetThumbprint = input.Thumbprint
+
 	// Discard anything other than these URL fields for the target
-	conf.Target = (&url.URL{
+	stripped := &url.URL{
 		Scheme: u.Scheme,
 		Host:   u.Host,
 		Path:   u.Path,
-	}).String()
+	}
+	conf.Target = stripped.String()
 
-	conf.Username = u.User.Username()
-	conf.Token, _ = u.User.Password()
-	conf.TargetThumbprint = input.Thumbprint
+	// validate that the provided operations credentials are valid
+	stripped.Path = "/sdk"
+
+	var soapClient *soap.Client
+
+	if input.Thumbprint != "" {
+		// if any thumprint is specified, then object if there's a mismatch
+		soapClient = soap.NewClient(stripped, false)
+		soapClient.SetThumbprint(stripped.Host, input.Thumbprint)
+	} else {
+		soapClient = soap.NewClient(stripped, input.Force)
+	}
+	soapClient.UserAgent = "vice-validator"
+
+	vimClient, err := vim25.NewClient(ctx, soapClient)
+	if err != nil {
+		v.NoteIssue(fmt.Errorf("Failed to create client for validaiton of operations credentials: %s", err))
+		return
+	}
+
+	client := &govmomi.Client{
+		Client:         vimClient,
+		SessionManager: vmomisession.NewManager(vimClient),
+	}
+
+	err = client.Login(ctx, url.UserPassword(input.OpsUser, input.OpsPassword))
+	if err != nil {
+		v.NoteIssue(fmt.Errorf("Failed to validate operations credentials: %s", err))
+		return
+	}
+	client.Logout(ctx)
```

Comment:
feels like this is a new validation for ops user, instead of target. And will it be cleaner to leverage session.connect?
I know that we're going to remove session eventually, but the login in session might be still useful.
my 2 cents.